.travis.yml

```diff
@@ -1,19 +1,15 @@
-# see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
 
-# list any PHP version you want to test against
 php:
-  # using major version aliases
-
-  # aliased to 5.3.29
+  - 5.3.3
```

Comment:
Travis exclusively supports testing on php 5.3.3. We should probably make sure we're compatible with this version since quite a few people may be using it.